src/status_im/ui/screens/accounts/recover/events.cljs

```diff
@@ -17,11 +18,15 @@
 
 (re-frame/reg-fx
  ::recover-account-fx
- (fn [[passphrase password]]
+ (fn [[masked-passphrase password]]
    (status/recover-account
-    (string/trim passphrase)
+    (security/unmask masked-passphrase)
     password
-    #(re-frame/dispatch [:account-recovered % password]))))
+    (fn [result]
+      (let [data (-> (types/json->clj result)
+                     (update :mnemonic security/mask-data)
+                     (types/clj->json))]
```

Comment:
To be honest, I don't really like this part, but it's the easiest way of hiding mnemonic.
If we want to make code more clear, the options are the following:
1) hide this information from logs completely. in this case we simply need to transform json to clj and `dissoc :mnemonic`
2) make `status/recover-account` to return everything except mnemonic
2)

Reply:
@pilu Any reason why mnemonic is returned from `recover-account` ?